scripts/calculate-hashes.ts

```diff
@@ -32,6 +32,20 @@ const findDirsEndingWith =
     }
   };
 
+const findFilesEndingWith =
```

Comment:
It seems that the only use-case of it in could be replaced with simply two parameter function (it will be more readable)

Reply:
fixed